css/src/lib.rs

```diff
@@ -109,22 +105,19 @@ const RESERVED3_LEN: usize = 32;
 const RESERVED4_LEN: usize = 12;
 
 /// The SIGSTRUCT structure
-#[repr(C, packed)]
-#[derive(Clone)]
+#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 pub struct Signature {
     /// The byte stream 0x06000000E10000000000010000000000
     header: [u8; HEADER_LEN],
     /// Intel enclaves will use 0x00008086, other enclaves will use 0x00000000
-    vendor: [u8; VENDOR_LEN],
+    vendor: EnclaveVendor,
     /// The build date of the enclave, as an 8-decimal date (YYYYMMDD) stored as
     /// little-endian bytes
     date: [u8; DATE_LEN],
     /// The byte stream 0x01010000600000006000000001000000
     header2: [u8; HEADER2_LEN],
     /// Data defined by the signature software
     swdefined: [u8; SWDEFINED_LEN],
```

Comment:
Again since this is no longer repr C packed the reserved bytes seemed unecessary to keep around